src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+    private Random random;
+
+    public Lottery(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+        this.random = new Random();
+    }
+
+    public Ball getRandomBall() {
+        Color randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(101);
```

Comment:
Make 101 constant
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/252559fd-a580-4616-b3b1-64c6bd40b146)

Reply:
ayyy, didn't notice that number, changing it :D Thank you for review